test/OpenSpaceToolkit/Astrodynamics/Flight/Profile.test.cpp

```diff
@@ -34,99 +37,101 @@
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-TEST (OpenSpaceToolkit_Astrodynamics_Flight_Profile, Constructor)
-{
+using ostk::astro::trajectory::Orbit ;
+using ostk::astro::flight::Profile ;
 
-    using ostk::core::types::Shared ;
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-    using ostk::physics::time::Instant ;
-    using ostk::physics::coord::Transform ;
-    using ostk::physics::coord::Frame ;
-    using ostk::physics::coord::frame::provider::Dynamic ;
+class OpenSpaceToolkit_Astrodynamics_Flight_Profile : public ::testing::Test {
 
-    using ostk::astro::flight::Profile ;
+    protected:
 
-    {
-
-        const Dynamic dynamicTransformProvider =
+        void                    SetUp                                       ( ) override
```

Comment:
I really like this "fixture" pattern! Could be really useful in a lot of places in the codebase :)